core/transactor/transactor.go

```diff
@@ -101,6 +109,24 @@ func (t *Transactor) FetchFees() (Fees, error) {
 	return f, err
 }
 
+// TopUp requests a myst topup for testing purposes.
+func (t *Transactor) TopUp(id string) error {
+	channelAddress, err := pc.GenerateChannelAddress(id, t.registryAddress, t.channelImplementation)
+	if err != nil {
+		return errors.Wrap(err, "failed to calculate channel address")
+	}
+
+	payload := TopUpRequest{
+		Identity: channelAddress,
+	}
+
+	req, err := requests.NewPostRequest(t.endpointAddress, "fee/topup", payload)
+	if err != nil {
+		return errors.Wrap(err, "identity request to Transactor failed")
```

Comment:
This error message sounds like it failed to do http request, but it only failed to create request object. Maybe

```go
return errors.Wrap(err, "failed to construct TopUp request")
```